### PR TITLE
buddhi: AMP setup and traffic generation tool

### PR DESCRIPTION
**Pull request by eguzki**
buddhi: AMP setup and traffic generation tool

### CODE REVIEW

**Comment by eguzki** [comment]
Cannot be merged until pisoni is being published in rubygems

File: buddhi/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Buddhi - 3Scale AMP service setup and traffic generation tool
+
+Responsabilitites:
```
Comment (davidor) [inline buddhi/README.md]: typo

File: buddhi/amp-toolkit-buddhi.gemspec [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  spec.bindir        = 'exe'
+  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  spec.require_paths = ['lib']
+
+  spec.add_development_dependency 'bundler', '~> 1.14'
+  spec.add_development_dependency 'rake', '~> 10.0'
+  spec.add_development_dependency 'rspec', '~> 3.5'
+  spec.add_development_dependency 'rspec-collection_matchers'
+  spec.add_development_dependency 'simplecov'
+  spec.add_dependency 'net-http-persistent', '< 3' # 3.0 is incompatible
```
[the remark this answers is not in the document]
Comment (eguzki) [reply buddhi/amp-toolkit-buddhi.gemspec]: https://github.com/drbrain/net-http-persistent/issues/80

File: buddhi/README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Buddhi - 3Scale AMP service setup and traffic generation tool
 
-Responsabilitites:
+Responsibilitites:
```
Comment (davidor) [inline buddhi/README.md]: typo :eyes:

**Comment by davidor** [comment]
I think the `Gemfile.lock` should be included too because this is a program and not a gem that will be required from other programs.

**Comment by davidor** [comment]
The code itself was already peer-reviewed when this repo was private, so I think you can go ahead and merge :+1: 

We'll need to set up a CI system like Travis or CircleCI, in this PR or a future one. As you prefer.

**Pull request merged**